sql/select_order_limit.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: Dan Harrison (daniel.harrison@gmail.com)
+
+package sql
+
+import (
+	"github.com/cockroachdb/cockroach/roachpb"
+	"github.com/cockroachdb/cockroach/sql/parser"
+)
+
+func (p *planner) SelectOrderLimit(n *parser.SelectOrderLimit, autoCommit bool) (planNode, *roachpb.Error) {
```

Comment:
I would put this in an existing file (maybe select.go) given that it's just one function.